lib/core/src/sdk.rs

```diff
@@ -216,6 +223,11 @@ impl LiquidSdk {
         let buy_bitcoin_service =
             Arc::new(BuyBitcoinService::new(config.clone(), breez_server.clone()));
 
+        let dns_resolver = Arc::new(TokioAsyncResolver::tokio(
+            ResolverConfig::default(),
+            ResolverOpts::default(),
```

Comment:
Did some more digging, we have to make sure we're using DNSSEC, so we need to set the validate field to true here (disabled by default)